src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  src: url("fonts/Avenir-Book.ttf");
+  font-family: "Avenir";
+  font-style: normal;
```

Comment:
Remove redundant default property

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/140645384/7742940c-079c-4ef6-9897-2d370f8d82b3)

Reply:
have done as in checklist